grammars/pact.cson

```diff
@@ -277,7 +277,7 @@
     'patterns': [
       {
         'begin': '\''
-        'end': '(?=\\s|[,:\"\'\.])'
+        'end': '(?=\\s|[],:\"\'\.)])'
```

Comment:
This looks like it ends the character class too soon? Doesn't that close bracket have to be escaped? Is `,` or `:` still working as a terminator?